app/styles/main.css

```diff
@@ -19,6 +19,11 @@
   background-image: none;
 }
 
+.faq b {
```

Comment:
wouldn't `.faq label[for="search"]` do the trick, to avoid the additional `<b>` tag?